viewing_party/main.py

```diff
@@ -0,0 +1,237 @@
+
+def create_movie(title, genre, rating):
+    if title == None:
+        return None
+    elif genre == None:
+        return None
+    elif rating == None: 
+        return None
+    else:
+        movie_dict = {
+            'title': title, 
+            'genre': genre, 
+            'rating': rating
+        } 
+    print("movie dictionary:",movie_dict)
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+
+    '''
+    the value of `movie` will be a dictionary in this format:
+    
+      {
+        "title": "Title A",
+        "genre": "Horror",
+        "rating": 3.5
+      }
+    '''
+    #the value of `user_data` will be a dictionary \n
+    #  with a key `"watched"`, and a value `[]`
+    user_data["watched"] = []
```

Comment:
This clears out any items that were already on the watched list. Unfortunately we didn't have a test to catch this issue, but its still good to be on the lookout for bugs like this.